dependency-check-suppressions.xml

```diff
@@ -22,4 +22,11 @@
         <gav regex="true">^org\.jetbrains\.kotlin:*:.*$</gav>
         <cve>CVE-2020-15824</cve>
     </suppress>
+    <suppress>
```

Comment:
Is there no fix available for this? We shouldn't really be suppressing vulnerabilities unless no other option. Try rebasing with master as we fixed a few recently.

Reply:
Rebased from master and removed suppression but its failing dependencyCheck again
https://build.platform.hmcts.net/view/SSCS/job/HMCTS_SSCS/job/sscs-tribunals-case-api/view/change-requests/job/PR-1239/86/artifact/build/reports/dependency-check-report.html